entrypoint.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 
-yes n | cp -i /www/config.yml.dist /www/config.yml
-while true; do echo n; done | cp -Ri /app/dist/www/assets /www/assets 2>/dev/null
+while true; do echo n; done | cp -Ri /www/default-assets/* /www/assets/
+
```

Comment:
We need a way to launch a one of the default configs
You might want to swap out for **config.yml.dist.sample-sui** depending on what you want as the default page when a user starts on the container for the first time.

```suggestion
# This makes sure that at least the default config is available.
yes n | cp -i /www/assets/config.yml.dist /www/assets/config.yml
```